README.md

```diff
@@ -87,6 +88,19 @@ git checkout -b issue-329
 
 Now, you can suggest contributions, make necessary changes to existing files, or add new files.
 
+## Identifying Documentation Source
+
+Some docs under [open-horizon.github.io/docs](https://github.com/open-horizon/open-horizon.github.io/tree/master/docs) can have its source in different repository.
```

Comment:
Instead of saying "some docs", I propose, "Some markdown pages"  under [] 
"can have their source markdown in other repositories in the Open Horizon GitHub organization."

Reply:
addressed